supervised-learning/knn_classifier.md

```diff
@@ -211,12 +189,7 @@ maps each word to the number of times that word appears:
 'birthday': 1, 'blend': 1, 'bloody': 1, 'bread': 1, 'breakfast': 1, ... }
 ```
 
-The `weighted_jaccard` distance measures the difference between two sets,
-weighted by the counts of each elements. To combine this output with the numeric
-distance we used above, we specify a **composite distance**. Each element in
-this list includes a list (or tuple) of feature names, a standard distance
-function name, and a numeric weight. The weight on each component can be
-adjusted to produce the same effect as normalizing features.
+The `weighted_jaccard` distance measures the difference between two sets, weighted by the counts of each elements. To combine this output with the numeric distance we used above, we specify a **composite distance**. Each element in this list includes a list (or tuple) of feature names, a standard distance function name, and a numeric weight. The weight on each component can be adjusted to produce the same effect as normalizing features.
```

Comment:
Since this distance is not standard, it's worth linking to the API docs or including the equation inline.